play-json/shared/src/main/scala/play/api/libs/json/Reads.scala

```diff
@@ -425,6 +400,13 @@ trait DefaultReads extends LowPriorityDefaultReads {
     }
   }
 
+  /**
+   * Deserializer for BigInt
+   */
+  implicit object BigIntReads extends Reads[BigInt] {
```

Comment:
```suggestion
  implicit val bigIntReads: Reads[BigInt] = BigIntegerReads.map(BigInt(_))
```

Reply:
This will not pass binary compatibility check.